dependencies/ACE/CMakeLists.txt

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-project( ACE VERSION 6.5.9 LANGUAGES NONE )
+project( ACE VERSION 6.5.12 LANGUAGES NONE )
```

Comment:
Ugh - so disappointing that we have to take another update to this one before we can get rid of it altogether.

Reply:
Well, it might not still work on macOS arm chips because BigSur support was added after 6.5.12 was cut https://github.com/DOCGroup/ACE_TAO/commit/61338c70ac750777c0b8e34f2573c4800e7de30b